www.example.com/db/openid-store/associations/http-localhost-Sve98kiqQxHfzN8CdZK0Uuu1aIs-JbRDRpXM2E2.VCy3cFsI5CJWF2k

```diff
@@ -0,0 +1,6 @@
+version:2
```

Comment:
Same for this file

Reply:
Thanks for all your help and for my first PR 👍